packages/mobile/src/pages/ScreensAuthorization/Authorization/Authorization.tsx

```diff
@@ -7,6 +7,8 @@ import { CustomButton } from '../../../components/Button';
 import { Divider } from '../../../components/Divider';
 import styles from './styles';
 const autorizationBgImg = require('./../../../assets/img/bg-home.jpg');
+const iconSrcGoogle = './../../../assets/icons/google.png';
+const iconSrcFb = './../../../assets/icons/fb.png';
```

Comment:
Why not requiring them here as you do `authorizationBgImg`?

Reply:
I will fix this, thanks.